include/circt/Conversion/SCFToCalyx.h

```diff
@@ -21,10 +21,16 @@
 namespace circt {
 
 namespace scfToCalyx {
-// If this attribute is set as a FuncOp argument or result attribute, it will be
-// used as the Calyx port name.
+// If this string attribute is set as a FuncOp argument or result attribute, it
+// will be used as the Calyx port name.
 static constexpr std::string_view sPortNameAttr = "calyx.port_name";
 
+// If this bool attribute is set as a FuncOp argument of type MemRefType,
+// the memory will be considered as having sequential reads.
+// This adds a "read_en" signal to the external memory interface, driven by
+// memref::LoadOp.
+static constexpr std::string_view sSequentialReads = "calyx.sequential_reads";
```

Comment:
does std::string_view's static constructor result in a blob in the right segment or is there a function at startup?  LLVM has strong feelings about the latter.